packages/p2p/src/utils/websocket.js

```diff
@@ -159,4 +161,7 @@ const getModifiedResponse = response => {
     return modified_response;
 };
 
-export const subscribeWS = (request, cb) => ws.p2pSubscribe(request, response => cb(getModifiedResponse(response)));
+export const subscribeWS = (request, callbacks) =>
```

Comment:
can we do multiple execution in the callback itself instead of handling multiple callbacks in subscription

Reply:
I think this was implemented to allow a separation of concerns, i.e. in this case it's used to set advertiser info and chat info, they're unrelated to each other so multiple callbacks make sense imo.